.changes/unreleased/Breaking Changes-20231129-091921.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Breaking Changes
+body: Rm --dry-run flag from 'dbt deps --add-package', in favor of just 'dbt deps
```

Comment:
This change seems not so relevant to what you are doing in this PR?

Reply:
yea I'm not sure why this shows as a diff as the PR for this was merged last week?

Reply:
https://github.com/dbt-labs/dbt-core/pull/9169